index.d.ts

```diff
@@ -504,7 +504,12 @@ declare module "skygear/cloud" {
     options?: ConfigOptions
   ): Promise<any>;
 
-  export class CloudCodeContainer extends BaseContainer {}
+  export class CloudCodeContainer extends BaseContainer {
+    asUserId: string;
+    public auth: AuthContainer;
+    public publicDB: Database;
+    public pubsub: PubsubContainer;
```

Comment:
remove public modifier because it is the default modifier if no modifier is specified.